src/Dropdown/test/DropdownMenuSpec.js

```diff
@@ -234,6 +234,30 @@ describe('<Dropdown.Menu>', () => {
     assert.isNotNull(instance.querySelector('.rs-dropdown-item-submenu'));
   });
 
+  it('Should call onSelect callback with correct `eventKey` once', () => {
+    const selectedValues = [];
+
+    const instance = getDOMNode(
+      <DropdownMenu
+        onSelect={eventKey => {
+          selectedValues.push(eventKey);
+        }}
+      >
+        <DropdownItem eventKey={1}>1</DropdownItem>
+        <DropdownItem eventKey={2}>2</DropdownItem>
+        <DropdownItem eventKey={3}>3</DropdownItem>
+      </DropdownMenu>
+    );
+
+    act(() => {
+      Simulate.click(instance.querySelectorAll('[role^="menuitem"]')[2], {
+        bubbles: true
+      });
+    });
+
+    expect(selectedValues.length).to.be.eq(1);
```

Comment:
Please use `sinon-chai` assertions to assert function called and arguments. Example here https://github.com/rsuite/rsuite/blob/d5b67383b5/src/Dropdown/test/DropdownMenuItemSpec.js#L101

Reply:
> If there is any other way to use `sinon-chai` to handle that case please guide me to it.

There's a [property `.callCount`](https://sinonjs.org/releases/latest/spies/) recording how many times a spy functions has been called.